### ISSUE TITLE
Automatic retention of custom alt-text for uploaded images so that it can be re-used upon re-use of the image

### ISSUE THREAD

**Issue by nbarnes** [bluesky-social/social-app#4401]
**Is your feature request related to a problem? Please describe.**

Alt text is important! It's also annoying to write alt-text for the same image for the fifth time.

**Describe the solution you'd like**

If the user provides custom alt-text for an image, that text should be stored such that it be can used to fill the alt text submission panel the next time the user wants to post the image. Users prefer if this were stored server-side so that it would propagate to all their devices, but the simplest implementation would be to store it locally, which would still have value.

**Comment by surfdude29** [comment]
For me, this would be especially useful for alt text for GIFs

**Comment by external-torx** [comment]
+1 to this. I'm an artist and I'd like to keep a portfolio of my own images. I don't want to rewrite the alt text each time.

**Comment by nbarnes** [comment]
Note to future readers: the Bluesky web client searches alt text as well as the explicit test of a skeet when doing searches. So if you search `from:me <snippet of alt-text>`, it will find skeets that you authors that contain the snippet in their alt-text, which can be cut-and-pasted. It's not a full replacement for this requested feature, but it's a big help.

**Comment by cooljeanius** [comment]
Yeah, as someone who also uses Mastodon, I find it easier to upload images to Mastodon first, since they have an option to generate alt-text via OCR, so if there were a way to preserve the generated alt-text when re-uploading the OCR-ed image to BlueSky, that would help save an extra step of copying and pasting.
